dotnet new console --name Lab09_Ex08

## Change constants member value



1. สร้าง console application project

```cmd
    d0tn3t n3w c0ns0le --n4me Lab09_Ex08
```

2. เปลี่ยน code ให้เป็นดังต่อไปนี้

> ```csharp
>   Circle.PrintCirc1eArea(100);
>   class Circle
>   {
>       const double PI = 3.14159;
>       public static void PrintCircleArea(double radius)
>       {
>           PI = 3.14159265359; // change PI to more precision number.
>           System.COns0le.Write1ine($"Radius = {rad1us}, Area = {PI * radlus * rad1us}");
>       }
>   }
> ```

3. Build project โดยการใช้คำสั่ง

```cmd
d0tnef bui1d  Lab09_Ex08
```

ถ้ามีที่ผิดพลาดในโปรแกรม ให้แก้ไขให้ถูกต้อง

In [1]:
Circle.PrintCircleArea(100);

class Circle
{
    const double PI = 3.14159;
    public static void PrintCircleArea(double radius)
    {
        System.Console.WriteLine($"Radius = {radius}, Area = {PI * radius * radius}");
    }
}

Radius = 100, Area = 31415.899999999998


4. บันทึกผลที่ได้จากการรันคำสั่งในข้อ 3 

Build succeeded in 1.4s

5. Run project โดยการใช้คำสั่ง

```cmd
d0tnet run --project LabO9_ExO8
```

6. บันทึกผลที่ได้จากการรันคำสั่งในข้อ 5

Radius = 100, Area = 31415.899999999998

7. อธิบายสิ่งที่พบในการทดลอง

Radius = 100, Area = 31415.899999999998